### PR TITLE
feat(metrics):Ui for investigation rules

### PR DESCRIPTION
**Pull request by RaduW**
This PR adds a basic UI for the creating investigation Rules

Resolves:https://github.com/getsentry/sentry/issues/56864

### CODE REVIEW

File: static/app/views/discover/table/tableActions.tsx [hunk 1]
```diff
@@ -148,18 +151,25 @@ function FeatureWrapper(props: FeatureWrapperProps) {
...
-      <FeatureWrapper {...props} key="edit">
-        {renderEditButton}
-      </FeatureWrapper>
-      <FeatureWrapper {...props} key="download">
-        {renderDownloadButton}
-      </FeatureWrapper>
-      {renderSummaryButton(props)}
-    </Fragment>
+    <div className="table-actions">
+      <Fragment>
```
Comment (matejminar) [inline static/app/views/discover/table/tableActions.tsx:158]: Do we need the className?
I believe Fragment can be removed now that you wrapped it in div.

File: static/app/views/discover/table/tableActions.tsx [hunk 2]
```diff
@@ -148,18 +151,25 @@ function FeatureWrapper(props: FeatureWrapperProps) {
...
+        <InvestigationRuleCreation
+          {...investigationRuleProps}
```
Comment (matejminar) [inline static/app/views/discover/table/tableActions.tsx:160]: Should we just `{...props} numSamples={props.tableData?.data?.length}`?

File: static/app/components/investigationRule.tsx [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+  }
+  if (request.isLoading) {
+    return null;
+  }
+
+  if (request.error !== null) {
+    const errorResponse = t('Unable to fetch custom performance metrics');
+    addErrorMessage(errorResponse);
+    return null;
+  }
```
Comment (matejminar) [inline static/app/components/investigationRule.tsx:166]: This message seems like a bad copypaste.

File: static/app/components/dynamicSampling/investigationRule.tsx [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+      title={tct(
+        'We can find more events that match your search query by adjusting your retention priorities for an hour, increasing the odds of getting matching events. [link:Learn more.]',
+        // TODO find out where this link is pointing to
+        {
+          link: <ExternalLink href="https://docs.sentry.io" />,
+        }
+      )}
+    >
+      <Button
+        size="sm"
```
Comment (matejminar) [inline static/app/components/dynamicSampling/investigationRule.tsx:211]: Discover is using "sm", performance needs "xs". Let's add a `buttonProps` prop to the InvestigationRule component that will  explode them inside this button

**Review by matejminar — APPROVED** [review]

File: static/app/components/dynamicSampling/investigationRule.tsx (hunk 4 above)
Comment (RaduW) [reply static/app/components/dynamicSampling/investigationRule.tsx]: Done

**Pull request merged**